src/Configuration/ValueObject/ExportDatabaseConfig.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Configuration\ValueObject;
+
+use Keboola\DbExtractorConfig\Exception\PropertyNotSetException;
+
+class ExportDatabaseConfig
+{
+
+    private ?SSLConnectionConfig $sslConnectionConfig;
+
+    private string $host;
+
+    private ?int $port;
+
+    private string $username;
+
+    private string $password;
+
+    private ?string $database;
+
+    private ?string $schema;
+
+    public static function fromArray(array $data): self
+    {
+        if (!$data['host']) {
```

Comment:
K tomu mam 2 poznamky:
- Nevaliduje to ze kluc neexistuje (to by tam muselo byt `empty/isset`) ... validuje to iba `null` a empty string
- Ked som robil refactoring na value objecty tak Tomas mal k tomu taku pripomienku, ... ze toto ma zvalidovat `ConfigDefinition` a nema to byt validovane duplicitne ... tam sme sa dohodli, ze teda tie podmienky vyhodim .... takze ani tieto by tu nemali byt (vid ostatne `fromArray`).... ak si na to uz nespominas, tak mozem najst kde sa to riesilo, ... ale asi budes vedel o com hovorim

Reply:
jj když to říkáš tak si na to vzpomínám => odsud to teda vyhodím

Reply:
je to pryč